src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -3188,6 +3204,9 @@ subroutine internal_tides_init(Time, G, GV, US, param_file, diag, CS)
   call get_param(param_file, mdl, "INTERNAL_TIDE_ANGLES", num_angle, &
                  "The number of angular resolution bands for the internal "//&
                  "tide calculations.", default=24)
+  call get_param(param_file, mdl, "DT_ITIDES", CS%dt_itides, &
+                 "The timestep for internal tides ray-tracing scheme", &
+                 units="s", default=3600., scale=US%s_to_T)
```

Comment:
Because the appropriate timestep is based on a CFL condition that is a function of the stratification and resolution, there should not be a hard-coded default value.  One option might be to set the default to match the baroclinic `DT`.  Another might be to make the default -1, and then use a single subcycle at about line 363 when `CS%dt_itides` is negative.